.changeset/config.json

```diff
@@ -11,6 +11,7 @@
   "updateInternalDependencies": "patch",
   "ignore": [],
   "___experimentalUnsafeOptions_WILL_CHANGE_IN_PATCH": {
+    "updateInternalDependents": "always",
```

Comment:
Might help bump dependent packages if there dependencies have changed ([docs](https://github.com/changesets/changesets/blob/e4204820cdb93d22a3c7a27f452a0ab30db6369a/docs/experimental-options.md#updateinternaldependents-type-out-of-range--always)).